File: Reqnroll/Chapter14/14.2-DefaultInstancePattern/After/WIMP.Specs/Support/DomainDefaults.cs
using WIMP.App.Models;

namespace WIMP.Specs.Support;

public static class DomainDefaults
{
    public const string PizzaName = "Margherita";
    public const string PizzaSize = "Medium";
    public const PizzaStyle PizzaStyle = App.Models.PizzaStyle.Regular;

    public static PizzaItem PizzaItemDefaultInstance()
    {
        return new PizzaItem(PizzaName, PizzaSize, PizzaStyle);
    }

    public const string CustomerName = "Rebecca";
    public const string CustomerEmail = "becca@galaxy.uni";
    public const string DeliveryAddress = "2-4 Waterloo Pl, Edinburgh EH1 3EG";
    public const string AltDeliveryAddress = "18 Holyrood Park Rd, Edinburgh EH16 5AY";
}

File: CucumberJVM/Chapter14/14.2-DefaultInstancePattern/Before/src/test/java/com/wimp/app/specs/support/DomainDefaults.java
package com.wimp.app.specs.support;

import com.wimp.app.models.PizzaStyle;

public final class DomainDefaults {
    public static final String CUSTOMER_NAME = "Rebecca";
    public static final String CUSTOMER_EMAIL = "rebecca@example.com";
    public static final String CUSTOMER_ADDRESS = "2850 Piedmont Ave NE, Apt 3C, Atlanta, GA 30308";
    public static final String ALT_DELIVERY_ADDRESS = "191 Peachtree St NE, Suite 6200, Atlanta, GA 30303";

    public static final String PIZZA_NAME = "Margherita";
    public static final String PIZZA_SIZE = "Medium";
    public static final PizzaStyle PIZZA_STYLE = PizzaStyle.REGULAR;

    private DomainDefaults() {
    }
}
